react/react-training/src/app/components/AppRoutes/index.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import Game from '@screens/Game';
+import Login from '@screens/Login';
+
+export default class AppRoutes extends Component {
+  render() {
+    return (
+      <Router>
+        <div>
```

Comment:
Change this for [`Switch`](https://reacttraining.com/react-router/core/api/Switch)